images/structure.md

```diff
@@ -43,3 +43,36 @@ USER coder
 Your users can then create descendant images. These contain all of the original
 tooling and configuration installed onto the base image and the new
 customizations added by the users.
+
+## Coder assets
+
+Coder inserts static assets into each workspace, including:
+
+- code-server
+- JetBrains Projector
+- Coder CLI
+
+These assets are installed into the `/var/tmp/coder` directory of each
+workspace. You do not need to include these statics assets in your custom
+images. However, the following software are **required** when you build custom
```

Comment:
purely a grammatical question - is this supposed to be `the following software are` or `the following software is`?

Reply:
"is" I think, @khorne3

Reply:
I actually Googled this because I didn't know -- software is both singular and plural, but I ended up choosing plural because the list has 2+ items